packages/nextjs/src/common/FronteggRouterBase.tsx

```diff
@@ -25,9 +25,10 @@ export function FronteggRouterBase(props: FronteggRouterBaseProps) {
     if (!app) {
       return;
     }
+
     const pathname = `/${pathArr.join('/')}`;
     const routesObj = {
-      ...authInitialState.routes,
+      ...defaultFronteggRoutes,
```

Comment:
why?

Reply:
no more access to authInitialState. there is new export that have the default frontegg routes instead of getting them from authInitialState.routes